okpt/io/config/parsers/nmslib.py

```diff
@@ -19,12 +19,34 @@
 Classes:
     NmslibParser: NMSLIB config parser.
 """
+from dataclasses import dataclass
 from io import TextIOWrapper
-from typing import Any, Dict
 
 from okpt.io.config.parsers import base
 
 
+@dataclass
+class MethodParametersConfig:
+    ef_construction: int
+    m: int
+
+
+@dataclass
+class MethodConfig:
+    name: str
+    space_type: str
+    parameters: MethodParametersConfig
+
+
+@dataclass
+class NmslibConfig:
+    method: MethodConfig
+    ef_search: int
```

Comment:
Is post and ef_search not a method parameter?

Reply:
You're right, I've updated the method schema